tests/bench/coremark/Kconfig

```diff
@@ -7,12 +7,21 @@ menuconfig COREMARK
 
 if COREMARK
 
+config COREMARK_FORCE_VERY_QUICK
+	bool "Use minimal data size and iteration count to run quickly"
+	default false
```

Comment:
default n?
Or you should just omit this line as the default default is n.

Reply:
> default n?
> Or you should just omit this line as the default default is n.

fixed